fmn/consumer/backend.py

```diff
@@ -175,9 +178,11 @@ def read(queue_object):
                 idx = recipient['filter_id']
                 fltr = session.query(fmn.lib.models.Filter).get(idx)
                 fltr.fired(session)
-        except:
-            # If anything happens, put the message back in the queue and bail
-            ch.cancel()
+        except RuntimeError:
+            yield ch.basic_nack(delivery_tag=method.delivery_tag)
+            return
+        except smtplib.SMTPSenderRefused:
```

Comment:
What i have noticed with sse, is that when it crashes and I try to load the backend again it will crash on the same message. So in the sse handle method I added `import pdb pdb.set_trace()` and I walk through the program. pdb is actually nice because it tells you more info of why it crashes then the console would.